### PR TITLE
Added `no-duplicate-imports` rule

### PR DESCRIPTION
**Pull request by GermanBluefox**
https://eslint.org/docs/latest/rules/no-duplicate-imports

### CODE REVIEW

**Comment by foxriver76** [comment]
Is it still allowed to have a separate line for all type exports?

**Comment by foxriver76** [comment]
I personally use it sometimes when having too many type imports as the compiler removes `import type { ... } from 'xy'`syntax completely it is not bad to have it in src. I mean it is fine for me but at some point we should have a final base set of ESLint rules and not modify it to death for personal preferences.

File: eslint.config.mjs [hunk 1]
```diff
@@ -44,6 +44,7 @@ const generalRules = {
     // https://eslint.org/docs/latest/rules/prefer-template
     // Enforce the use of template literals instead of string concatenation: "Hello, " + name + "!" => `Hello, ${name}!`
     'prefer-template': 'error',
+    '@/no-duplicate-imports': 'error',
```
Comment (foxriver76) [inline eslint.config.mjs:47]: should be

```suggestion
    'no-duplicate-imports': 'error',
```

**Review by foxriver76 — CHANGES_REQUESTED** [review]

**Comment by GermanBluefox** [comment]
> I personally use it sometimes when having too many type imports as the compiler removes `import type { ... } from 'xy'`syntax completely it is not bad to have it in src. I mean it is fine for me but at some point we should have a final base set of ESLint rules and not modify it to death for personal preferences.

Early we used airbnb rules, that was based on Google JS Style. airbnb are not more maintained any more because of eslint 9
All these rules we had there and even much more. If airbnb would be maintained again, I would use them with a great pleasure.

This PR is just to get back what we had....

**Comment by foxriver76** [comment]
> > I personally use it sometimes when having too many type imports as the compiler removes `import type { ... } from 'xy'`syntax completely it is not bad to have it in src. I mean it is fine for me but at some point we should have a final base set of ESLint rules and not modify it to death for personal preferences.
> 
> Early we used airbnb rules, that was based on Google JS Style. airbnb are not more maintained any more because of eslint 9 All these rules we had there and even much more. If airbnb would be maintained again, I would use them with a great pleasure.
> 
> This PR is just to get back what we had....

All good, just wanted to point it out.

**Review by foxriver76 — APPROVED** [review]

**Pull request merged**